test/tests/memcached-basics/run.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# "/dev/tcp/..." (https://www.gnu.org/software/bash/manual/bashref.html#Redirections-1)
+clientImage='bash'
+
+cname="memcached-container-$RANDOM-$RANDOM"
+cid="$(docker run -d --name "$cname" "$image")"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+# usage: memcached-command end-regex command [extra-line ...]
+#    ie: memcached-command '^END$' 'stats'
+#        memcached-command '^((NOT_)?STORED|EXISTS|NOT_FOUND)$' 'set a 0 0 2' 'hi'
+memcached-command() {
+	local bashScript
+	read -r -d '' bashScript <<-'EOBASH' || :
+		endRegex="$1"; shift
+		exec 3<>/dev/tcp/memcached/11211
```

Comment:
This hacky logic using `/dev/tcp/...` is cute, but it turns out `busybox` includes `nc`, which closes the input and makes this _way_ easier to implement. :disappointed: :innocent:

ie:
```console
bash-4.4# echo -e 'get a\r' | nc memcached 11211
END
bash-4.4# 
```
(closes the input automatically at EOF, which tells memcached to close the output too, thus removing the need for my `timeout` and `endRegex` nonsense)